src/Components/Home/ListItem.js

```diff
@@ -10,16 +10,15 @@ function ListItem(prop) {
              <Row>
                 <Col>
                     {daysLeft == 0 ?  
-                        <Button variant="outline-dark" className="px-3"><i className="fas fa-shopping-cart"></i> Buy</Button>: 
+                        <Button variant="danger" className="px-3"><i className="fas fa-minus-circle"></i> Forgot</Button>: 
                         <span><i className="far fa-clock"></i> {daysLeft} days left</span>}
                 </Col>
                 <Col className="text-right">
-                    <Button variant="outline-dark"><i className="fas fa-pen" ></i></Button>
-                    <Button variant="outline-dark" style={{paddingLeft:'0.55em', paddingRight:'0.2em'}}>
+                    <Button variant="warning"><i className="fas fa-pen" ></i></Button>
+                    <Button variant="success" style={{paddingLeft:'0.55em', paddingRight:'0.2em', marginLeft:'0.3em'}}>
                         <i className="fas fa-thumbs-up"></i>
                         <sup><Badge variant="light">{likeCount}</Badge></sup>
                     </Button>
-                    <Button variant="outline-dark"><i className="fas fa-trash-alt" ></i></Button>
```

Comment:
i don't remember why are we removing this buttton. are we moving it to edit view?

Reply:
I think it would be better hidden a bit more, in the edit page is a good place.